### ISSUE TITLE
How to deal with when output channel number C‘ are not equal to input channel number C?

### ISSUE THREAD

**Issue by wenhangyu** [backseason/SCNet#13]
when output channel number C‘ are not equal to input channel number C，there is different channel numbers between identity with self.k2(x). So how to deal with this problem? The code as follows:

out = torch.sigmoid(torch.add(identity, F.interpolate(self.k2(x), identity.size()[2:])))

**Comment by backseason** [comment]
You may 1) set the output and input channels of `self.k2()` to be identical, or 2) replace the `identity` operation with a simple conv1x1 layer.
